core/bin/external_node/src/config/mod.rs

```diff
@@ -932,6 +934,10 @@ pub(crate) struct RequiredENConfig {
 }
 
 impl RequiredENConfig {
+    pub fn get_settlement_layer_id(&self) -> SLChainId {
```

Comment:
Nit: getters in Rust [usually don't have `get_` prefix](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter), so `settlement_layer_id` would look better.